spec/controllers/users/webauthn_setup_controller_spec.rb

```diff
@@ -172,6 +172,7 @@
       stub_analytics
       stub_sign_in(user)
       allow(IdentityConfig.store).to receive(:domain_name).and_return('localhost:3000')
+      request.host = 'localhost:3000'
```

Comment:
This is required to ensure that the webauthn domain name does not conflict with the `host` in the request, otherwise we get (and also because this isn't a request where we want to allow other hosts):

```
ActionController::Redirecting::UnsafeRedirectError:
       Unsafe redirect to "http://localhost/account", pass allow_other_host: true to redirect anyway.
```